src/Microsoft.Tye.Core/ApplicationFactory.cs

```diff
@@ -102,21 +102,19 @@ public static async Task<ApplicationBuilder> CreateAsync(OutputContext output, F
                     project2.Frameworks.Any(f => f.Name == "Microsoft.AspNetCore.App"))
                 {
                     // HTTP is the default binding
-                    var httpBinding = new BindingBuilder()
+                    service.Bindings.Add(new BindingBuilder()
                     {
                         AutoAssignPort = true,
                         Protocol = "http"
-                    };
-
-                    var httpsBinding = new BindingBuilder()
-                    {
-                        Name = "https",
-                        AutoAssignPort = true,
-                        Protocol = "https"
-                    };
+                    });
 
-                    service.Bindings.Add(httpBinding);
-                    service.Bindings.Add(httpsBinding);
+                    // TODO: Uncomment when the CI is unblocked
+                    //service.Bindings.Add(new BindingBuilder()
```

Comment:
What's the problem with this?

Reply:
dev cert broken on the CI (expired on windows and untrusted on osx/linux)

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=574074&view=ms.vss-test-web.build-test-results-tab&runId=18003516&paneView=attachments&resultId=100013